File: 7-kyu/disemvowel-trolls/disemvowel-trolls.py
def disemvowel(string_):
    res = ''
    for s in string_:
        if s == 'a' or s == 'e' or s == 'i' or s == 'o' or s == 'u' or s == 'A' or s == 'E' or s == 'I' or s == 'O' or s == 'U':
            continue
        res += str(s)
    return res

File: 7-kyu/disemvowel-trolls/disemvowel-trolls.go
package kata
​
func Disemvowel(comment string) string {
    res := []rune{} // use rune slice for efficient appending
​
    for _, s := range comment {
        switch s {
        case 'a', 'e', 'i', 'o', 'u', 'A', 'E', 'I', 'O', 'U':
            // skip vowels
            continue
        default:
            res = append(res, s)
        }
    }
​
    return string(res)
}
​